### ISSUE TITLE
and refined storage Incompatible

### ISSUE THREAD

**Issue by 710590662** [bafomdad/realfilingcabinet#33]
Minecraft 1.12.2 
Version 0.1.27
The file cabinet uses fluid upgrading after refined storage can only identify the first registered fluid, and then the registered fluid will not be identified.

**Comment by bafomdad** [comment]
after some investigating looking at other kinds of blocks that can store multiple different types of fluids within it, it would seem that refined storage is only able to see the first fluid in them (one such example being the smeltery controller/drain from Tinker's Construct)

how do you mean the registered fluid will not be identified?

**Comment by 710590662** [comment]
Yes, refined storage can only identify the first registered fluid, and then the registered fluid will not be identified.

**Comment by bafomdad** [comment]
hmm, I'm not quite sure what is going on with that..I saw it unregister the fluid from the tinker smeltery too. Breaking and replacing the external storage block fixed it for both, probably some kind of mod interaction thing.

if there are steps that can be followed to make this reproducible each time, it can be worth a look.
